cairo-contracts/packages/contracts/src/tests/client.cairo

```diff
@@ -58,8 +58,6 @@ fn test_update_comet_client_ok() {
     // Update Client
     // -----------------------------------------------------------
 
-    core.register_relayer(RELAYER());
```

Comment:
is this cleanup from the previous PR for client update?

Reply:
This is removed here, since I moved this relayer registration to a step higher where we setup contracts.
Because, by changes in this PR, in bunch of other tests it's required only authorized relayers be processing packets.